test/src/test/java/hudson/model/FileParameterValueTest.java

```diff
@@ -125,7 +131,7 @@ public void fileParameter_cannotCreateFile_outsideOfBuildFolder_SEC1424() throws
         assertThat(root.child("pwned").exists(), equalTo(false));
 
         // ensure also the file is not reachable by request
-        JenkinsRule.WebClient wc = j.createWebClient();
+        JenkinsRule.WebClient wc = getWebClient();
         wc.getOptions().setThrowExceptionOnFailingStatusCode(false);
     }
```

Comment:
(seems like it was missed to check the intent `// ensure also the file is not reachable by request`)
I will file a separate PR for completing this test (should be couple of lines only)